docs/src/examples/components/Dropdown/Variations/DropdownExamplePosition.shorthand.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import { Grid, Dropdown } from '@stardust-ui/react'
+
+const inputItems = ['Bruce Wayne', 'Natasha Romanoff', 'Steven Strange', 'Alfred Pennyworth']
+
+const DropdownArrowExample = props => {
+  const { position, align } = props
```

Comment:
Just wonder, will knobs more usable for this example?

Reply:
hmm.. we can but for Popup we don't do that for very similar example. I can make the change for both for consistency and use knobs , but which one do you recommend?

@layershifter

Reply:
I think that we can both, it will make them more understandable and compact I think

Reply:
that took longer than expected; reimplemented Dropdown and Popup examples related to position using select knobs